tests/test_notanorm.py

```diff
@@ -536,6 +562,12 @@ def test_timeout_rational(db_notmem):
     assert db.timeout < 60
 
 
+def test_db_cursor_can_exec(db):
+    # this really shouldn't be a guarantee, imo!
```

Comment:
:+1: would probably restrict us quite a bit if we added a "real" NoSQL backend (unless anyone has a real desire to support SQL-on-Mongo/Redis/Elastic).

Reply:
yeah, we rely on it in cvfs tho, so for now, it's a guarantee